stores/gameState.js

```diff
@@ -26,18 +26,48 @@ var state = {
 
 GameState.__onDispatch = function (payload) {
   switch(payload.actionType) {
-    case "GUESS_ADDED":
-      makeGuess(payload.guess);
-      break;
     case "NEXT_ITEM":
       advanceItem();
       break;
     case "SET_ITEM":
       setCurrentItem(payload.key);
       break;
+    case "GUESS_ADDED":
+      makeGuess(payload.guessType, payload.guess);
+      break;
   }
 };
 
+var makeGuess= function(guessType, answer) {
+  var correctAnswer;
+  if (guessType === "Full Name") {
+    correctAnswer = GameState.currentItem().name.toLowerCase();
```

Comment:
We should ultimately separate out First & Last name in the data, but for now, this is okay.